src/Dijkstra/AST/Core.agda

```diff
@@ -11,7 +11,6 @@ open import Data.Fin
 open import Data.Product using (_×_ ; _,_ ; proj₁ ; proj₂)
```

Comment:
curious, would it be good to have all `Dijkstra` modules import "Dijkstra"'s `Prelude` rather than individual stuff?